packages/hyperion-react-testapp/src/AutoLoggingWrapper.ts

```diff
@@ -1,174 +1,174 @@
-    flowletPublisher: {
-      channel
-    },
-    triggerFlowlet: {
-      enableReactMethodFlowlet: false,
-      enableFlowletConstructorTracking: false,
-    },
-    react: {
-      ReactModule: React as any,
-      IReactDOMModule,
-      IReactModule,
-      IJsxRuntimeModule,
-    },
-    surface: {
-      enableReactDomPropsExtension: false,
-    },
-    sessionFlowID: {
-      domain: 'localhost',
-      cookieName: 'axaxax',
-    },
-    elementText: {
-      updateText(elementText: ExtendedElementText, domSource) {
-        elementText.isExtended = true;
-        // console.log("Element Text ", elementText, domSource);
-      },
-    },
-    uiEventPublisher: {
-      uiEvents: [
-        {
-          eventName: 'click',
-          cacheElementReactInfo: true,
-          enableElementTextExtraction: true,
-          eventFilter: (domEvent) => domEvent.isTrusted
-        },
-        {
-          eventName: 'mousedown',
-          cacheElementReactInfo: true,
-          enableElementTextExtraction: false,
-          eventFilter: (domEvent) => domEvent.isTrusted
-        },
-        {
-          eventName: 'keydown',
-          cacheElementReactInfo: true,
-          interactableElementsOnly: false,
-          enableElementTextExtraction: false,
-          eventFilter: (domEvent) => domEvent.code === 'Enter',
-        },
-        {
-          eventName: 'keyup',
-          cacheElementReactInfo: true,
-          interactableElementsOnly: false,
-          enableElementTextExtraction: false,
-          eventFilter: (domEvent) => domEvent.code === 'Enter',
-        },
-        {
-          eventName: 'change',
-          cacheElementReactInfo: true,
-          enableElementTextExtraction: true,
-          interactableElementsOnly: false,
-        },
-        // {
-        //   eventName: 'mouseover',
-        //   cacheElementReactInfo: true,
-        //   interactableElementsOnly: false,
-        //   enableElementTextExtraction: true,
-        //   durationThresholdToEmitHoverEvent: 1000,
-        // },
-      ]
-    },
-    heartbeat: {
-      heartbeatInterval: 30 * 1000
-    },
-    surfaceMutationPublisher: {
-      cacheElementReactInfo: true,
-      enableElementTextExtraction: false,
-    },
-    surfaceVisibilityPublisher: {},
-    network: {
-      requestFilter: request => !/robots/.test(request.url.toString()),
-      requestUrlMarker: (request, params) => {
-        const flowlet = FlowletManager.top();
-        if (flowlet) {
-          params.set('flowlet', flowlet.getFullName());
-        }
-      }
-    },
-    domSnapshotPublisher: {
-      eventConfig: [
-        'al_ui_event',
-        'al_surface_visibility_event'
-      ]
-    }
-  });
-
-  console.log('AutoLogging.init options:', AutoLogging.getInitOptions());
-  // console.log('dsid', getDomainSessionID('localhost'));
-  // console.log('dsid', getDomainSessionID());
-  console.log('sfid', getSessionFlowID());
-
-}
+/**

```

Comment:
Not sure what is this file like this!  I try to figure out why it is considered changed.